### PR TITLE
Fix Secrets.succeed arguments in example

### PR DESCRIPTION
**Pull request by simplystuart**
Simple documentation fix for improving clarity

### CODE REVIEW

File: src/Pages/Secrets.elm [hunk 1]
```diff
@@ -20,7 +20,7 @@ And your StaticHttp request in your Elm code looks like this:
 
     StaticHttp.request
         (Secrets.succeed
-            (\githubToken stripeoken ->
+            (\apiKey bearer ->
```
Comment (dillonkearns) [inline src/Pages/Secrets.elm]:
```suggestion
            (\apiKey githubToken ->
```

File: src/Pages/Secrets.elm [hunk 2]
```diff
@@ -20,7 +20,7 @@ And your StaticHttp request in your Elm code looks like this:
...
                 { url = "https://api.github.com/repos/dillonkearns/elm-pages?apiKey=" ++ apiKey
                 , method = "GET"
                 , headers = [ ( "Authorization", "Bearer " ++ bearer ) ]
```
Comment (simplystuart) [inline src/Pages/Secrets.elm]: You would have to update the argument here then as well:
```suggestion
                , headers = [ ( "Authorization", "Bearer " ++ githubToken ) ]
```

File: src/Pages/Secrets.elm [hunk 3]
```diff
@@ -20,10 +20,10 @@ And your StaticHttp request in your Elm code looks like this:
 
     StaticHttp.request
         (Secrets.succeed
-            (\githubToken stripeoken ->
+            (\apiKey bearer ->
```
Comment (dillonkearns) [inline src/Pages/Secrets.elm]:
```suggestion
            (\apiKey githubToken ->
```

**Comment by dillonkearns** [comment]
Merged 👍 Thank you!

**Pull request merged**